### DISCUSSION TITLE
[Dúvida] Melhor maneira de se criar um portifólio/blog/currículo pessoal.

### DISCUSSION THREAD

**Discussion by lucasportolima** [frontendbr/forum#1411] [Discussão]
Olá a todos, estou querendo implementar um portifólio/blog/currículo pessoal onde eu possa criar artigos e publicações afim de repassar esse conhecimento para comunidade e ao mesmo tempo encaminhar para algum entrevistador/empresa para demonstrar meus conhecimentos/projetos.

### As minha dúvidas:

1- A stack de desenvolvimento que eu escolhi foi react + react native + nodejs. Escolhi pela minha experiência e por consegui desenvolver em multiplataformas agregando mais valor ao projeto. Alguém teria outra stack ou uma dica de alguma lib a usar?

2- Quais projetos colocar no portifólio? Todos ou apenas os melhores? Qual o nível de bom um projeto tem que está para entrar no portifólio? Nesses projetos eu colocaria um link pro git, prints do projeto ou gifs e videos de testes e2e?

3- Para esse tipo de projeto é melhor um design mais clean ou algo mais chamativo para impressionar?

4- Vocês tem alguns exemplos de projetos com esse objetivo que eu possa usar como inspiração? 

Obrigado a todos.

**Comment by gabriel-hahn** [comment]
Lucas, minha opinião sobre as dúvidas:

1 - Acredito que o melhor a fazer é tu focar nessa stack. Ter conhecimentos em outras linguagens e frameworks é sempre muito bem vindo, mas focar em construir projetos utilizando a stack que tu escolheu, além também explorar libs do universo dela (Redux, Express, Next e etc), vão te auxiliar bastante em tu ter bastante conhecimento no assunto, além de conseguir comprovar isso.

2 - Eu costumo colocar tudo que é significante para mim no GitHub, e também estou construindo um site pessoal para expor isso. Mantenho repos privados para projetos que possuem um código que vai me auxiliar, e deixo publico (e bem documentado) projetos de cursos/pessoais que acho legal mostrar, devido a ideia do projeto ou pela complexibilidade dele. Nos projetos desenvolvidos durante algum curso, sempre referencio o GitHub do professor.
Se tu for mencionar estes projetos em uma entrevista ou em algum outro local, tenta focar nos projetos mais complexos que tu conseguiu desenvolver e nos que tu tem maior domínio. Independente do grau de dificuldade do projeto, mostra aqueles que foram importantes para ti.

3 - Depende do projeto. Acredito que o importante é a estrutura, código limpo e testes (se for um projeto maior e que tenha oportunidade de upgrade. Acredito não valer a pena sair fazendo testes unitários para projetos bem pequenos e específicos, só pra dizer que sabe hehe). Se tu focar somente no Front-end, talvez ter um cuidado maior com visual dos projetos seja importante, pelo menos o básico.

4 - Não sei se é um bom exemplo, mas no meu GitHub tem alguns projetos que faço e os desenvolvidos em cursos. Procuro seguir uma estrutura de Readme que curto e tento deixar os projetos mais organizados possível.

**Comment by ir4nbala** [comment]
3 - No caso do Design, experimente ler um pouco sobre UX/UI, foque no seu publico alvo. Não adianta ter uma interface chamativa se não for intuitivo para o usuário, foque em dar a melhor experiência para o usuário.

**Comment by tarciozemel** [comment]
1 - Acho que tem que usar o que você sabe mais. Com o tempo, vai evoluindo/alterando conforme a necessidade.

2 - https://desenvolvimentoparaweb.com/miscelanea/o-que-colocar-no-portfolio/
   - https://www.youtube.com/watch?v=O0a5Ucd7y-A
   - https://desenvolvimentoparaweb.com/tag/portfolio/

3 - Melhor o que for mais condizente ao seu estilo/personalidade. Portfolios DEVEM transparecer esse tipo de coisa.

4 - Naqueles sites tipo Awwwards não falta inspiração.
